ccm_web/server/src/config.ts

```diff
@@ -43,12 +43,21 @@ export interface Config {
 
 const logger = baseLogger.child({ filePath: __filename })
 
-const isString = (v: unknown): v is string => typeof v === 'string'
+// Tests if an unknown value is a non-empty string
+const isString = (v: unknown): v is string => typeof v === 'string' && v.length > 0
+
+// Tests if an unknown value is a number and not NaN
 const isNumber = (v: unknown): v is number => typeof v === 'number' && !isNaN(v)
```

Comment:
I'm taking another look at these type guards; upon reflection, I'm not sure I should be adding more test conditions to them -- though it doesn't throw any errors. It might be clearer and more flexibile if the tests were separate. the `!isNan(v)` and `v.length > 0` tests can't be inferred from subsequent typing, I don't think. I'd like to do some refactoring. I will create a separate issue and link it to this PR.

Reply:
Okay, see #175 and https://github.com/tl-its-umich-edu/canvas-course-manager-next/pull/174/commits/b989b87ad10402883b945ad25c843e21fa92353b

Reply:
I like that you ended up separating the string type checking and length checking.